### ISSUE TITLE
euflag package incompatible

### ISSUE THREAD

**Issue by mbertucci47** [latex3/tagging-project#1587]
In addition to missing Alt text or a ToUnicode value, it produces parent-child warnings.
```tex
\DocumentMetadata{tagging=on}
\documentclass{article}

\usepackage{euflag}

\title{euflag tagging test}

\begin{document}

Some text \euflag

\end{document}
```
warns
```
Package tagpdf Warning: Parent-Child 'pdf2:P' --> 'pdf2:Part'.
(tagpdf)                Relation is not allowed! on line 12
(tagpdf)                struct 7, text --> struct 8, text-unit


Package tagpdf Warning: Parent-Child 'pdf2:P' --> 'pdf2:P'.
(tagpdf)                Relation is not allowed! on line 12
(tagpdf)                struct 8, text-unit --> struct 9, text


Package tagpdf Warning: Alternative text for graphic is missing.
(tagpdf)                Using 'picture environment' instead.
```

**Comment by u-fischer** [comment]
This here should work (the main problem was to get the bbox right, so that ngpdf can extract the image ;-)):

~~~~latex
\DocumentMetadata{uncompress,tagging=on,
% debug=BBox %to debug the BBox
}
\documentclass{article}

\usepackage{euflag}

\title{euflag tagging test}
\makeatletter
\newcommand\euflagname{EU flag}
\renewcommand{\euflag}[1][\EUF@baseline]{%
  {%
    \fboxsep0pt           
    \raisebox{#1}{%
      \colorbox{PantoneReflexBlue}{%        
        \vbox to1em{\UseTaggingSocket{para/off}%
          \hsize1.5em
          \parskip0pt
          \parindent0pt
          \centering
          \makestars
        }% end vbox
      }% end colorbox
    }% end raisebox    
  }% end encapsulation
}% end command

\renewcommand{\makestars}{%
  \color{PantoneYellow}%
  \setlength{\unitlength}{1em}
  \divide\unitlength by18   
  \begin{picture}[alt=\euflagname,
  tagging-setup={adjust-BBox=-0.6em -0.7em 0.6em 0pt}
  ](6,6)(-2,3.5)
    \put(6,0){\eustar}
    \put(5.196,3){\eustar}
    \put(3,5.196){\eustar}
    \put(0,6){\eustar}
    \put(-3,5.196){\eustar}
    \put(-5.196,3){\eustar}
    \put(-6,0){\eustar}
    \put(-5.196,-3){\eustar}
    \put(-3,-5.196){\eustar}
    \put(0,-6){\eustar}
    \put(3,-5.196){\eustar}
    \put(5.196,-3){\eustar}
  \end{picture}%
}
\begin{document}
{\huge\euflag}


\euflag

Some text \euflag

xxx\euflag[-1cm]

\end{document}
~~~~
